docs/user/guides/search-analytics.rst

```diff
@@ -1,6 +1,39 @@
-How To Use Search Analytics
+How to use search analytics
```

Comment:
This is a much better "how to use" article, because it gives a lot of context and ideas for how to analyze the data. Veering slightly into explanation, but I think it's actually a great how to.

Reply:
True that it tries to do a better job at giving _practical explanation_ to the use of what's visible on a particular page... but the explanation is centered around performing a specific task (i.e. the how-to title), so I hope it doesn't break our concept of the 4 axis.

@humitos you mentioned on another very "dull" step-by-step how-to that it barely provided value. The example was really good. I quoted your response [here](https://github.com/readthedocs/readthedocs.org/pull/9785#issuecomment-1361808732). For cases like this, does it make sense that we ask ourselves questions like:

* _Can this how-to contain recommendations and practical advice without breaking the how-to format?_
* _Can this how-to be expanded with relevant troubleshooting?_
* _Worst-case: Is this how-to describing a task that's so trivial and self-evident that we might as well remove it?_

Reply:
> * _Can this how-to contain recommendations and practical advice without breaking the how-to format?_
> * _Can this how-to be expanded with relevant troubleshooting?_
> * _Worst-case: Is this how-to describing a task that's so trivial and self-evident that we might as well remove it?_

I think these are great questions to ask. In particular, if we're representing something trivial, I think a small video/gif would likely do a lot more good for users. Perhaps something to try in the future. You can embed mp4/webp/etc. in webpages now as a gif really easily, and much smaller: https://www.smashingmagazine.com/2018/11/gif-to-video/#replace-animated-gifs-with-video-in-the-browser

Reply:
All added to the Style-guide PR :+1: #9840